views/standard/spec/columns.js

```diff
@@ -179,6 +203,18 @@ define(['jquery', 'appearanceUtils'], function($, utils) {
 
             var signatures = pull.qa_signatures;
 
+            if (pull.milestone) {
+               var due_date = pull.milestone.due_on &&
+                new Date(pull.milestone.due_on);
+
+               // If milestone is past due date.
+               if (due_date && due_date < new Date()) {
+                  score -= 2000;
+               } else {
+                  score -= 1000;
+               }
+            }
+
```

Comment:
@marczych there may be a way to add the repeated logic to a generic column sort method. I'll see if it's doable. 

dev_block :star:



> On Oct 19, 2015, at 12:02 PM, Marc Zych <notifications@github.com> wrote:
> 
> In views/standard/spec/columns.js:
> 
> > @@ -179,6 +203,18 @@ define(['jquery', 'appearanceUtils'], function($, utils) {
> >  
> >              var signatures = pull.qa_signatures;
> >  
> > +            if (pull.milestone) {
> > +               var due_date = pull.milestone.due_on &&
> > +                new Date(pull.milestone.due_on);
> > +
> > +               // If milestone is past due date.
> > +               if (due_date && due_date < new Date()) {
> > +                  score -= 2000;
> > +               } else {
> > +                  score -= 1000;
> > +               }
> > +            }
> > +
> Is there an easy way to reuse this logic rather than duplicating it a few times?
> 
> —
> Reply to this email directly or view it on GitHub.
>